### DISCUSSION TITLE
Error in Draft Workflow

### DISCUSSION THREAD

**Discussion by purva-thakre** [unitaryfund/mitiq#1990] [General]
Draft Github Actions workflow was failing for all versions with error `Error: Unable to process file command 'env' successfully.`. 

https://github.com/unitaryfund/mitiq/actions/runs/6054448774/job/16431867571 

The same builds pass when the PR is not a draft.

**Comment by natestemen** [comment]
From a quick look at this, it looks to me like the expected output of the command to set the `changed_directories` is a single line. However in the case of the failing PR (https://github.com/unitaryfund/mitiq/pull/1987), both the `pec` directory was changed and `mitiq/utils.py` as well. Hence the variable that is expected to be a single module is now

```
pec
utils.py
```

I'm not sure what a good solution to this is honestly.

This is my deduction from reading https://stackoverflow.com/questions/74353311/. @Aaron-Robertson might want to chime in if you see it differently.

**Reply by purva-thakre** [reply]
Yep, I found something about multiline strings in the documentation. I didn't know if this applied to the issue at hand which is why I created a discussion instead of an issue. 

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

**Comment by purva-thakre** [comment]
Closing this discussion as an issue has been created for it.
